plugins/obs-webrtc/webrtc-services-main.c

```diff
@@ -0,0 +1,37 @@
+#include "util/text-lookup.h"
```

Comment:
This secondary module should either be merged with the primary `webrtc.c` module or split into a new plugin directory, similar to rtmp-services. For now, I would lean towards merging the two modules.

Reply:
Fixed by 218663b7069b8fa0119e0fd3d5341e6f760881cd